README.md

```diff
@@ -98,6 +98,26 @@ rhc_repositories:
 A release to set for the system. Use `{"state":"absent"}` to actually unset the
 release set for the system.
 
+    rhc_proxy: {}
+
+The details of the proxy server to use for connecting:
+```yaml
+rhc_proxy:
+  hostname: "proxy-hostname"
+  port: 4321
+  username: "proxy-hostname"
+  password: "proxy-password"
+```
+- `hostname` is the hostname of the proxy server
+- `port` is the port to which connect to on the proxy server
+- `username` is the username to use for authenticating on the proxy server;
+  it can be not specified if the proxy server does not require authentication
+- `password` is the password to use for authenticating on the proxy server;
+  it can be not specified if the proxy server does not require authentication
+
+Please note that specifying an empty dictionary, i.e. `{}`, *resets* all the
+proxy configurations to empty (effectively disabling the proxy server).
```

Comment:
hmm - this means that `rhc_proxy: {}` has a different semantic meaning than `rhc_auth: {}` and `rhc_server: {}`?  That seems inconsistent

Reply:
The preferred method of doing "specifying this parameter to this value will erase/reset configuration" is to use the `state: absent` thing, just like `rhc_release`.  So perhaps
```
If you want to erase the proxy settings, specify

    rhc_proxy:
      state: absent
```

Reply:
oh, good idea, will switch to that approach

also note that `rhc_auth` and `rhc_proxy` are different anyway: you should be able to reset `rhc_proxy`, while "resetting" does not make sense for `rhc_auth`